ES/ESmy019.xml

```diff
@@ -99,117 +189,180 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                      </handNote>
-                     <handNote xml:id="h2" script="Ethiopic">Hand a (<locus from="2r" to="20r">2r-20r</locus>): round, irregular letters'
-                        shapes.<!--The names of this hands need to be replaced by appropriate refs.--></handNote>
-                     <handNote xml:id="h3" script="Ethiopic"> Hand b (<locus from="21r" to="48r">21r-48r</locus>): (<persName role="scribe" ref="PRS11077Zadengel">Za-Dǝngǝl</persName>), angular letters'
-                        shapes.<!--The names of this hands need to be replaced by appropriate refs.--></handNote>
-                  </handDesc>
-                  <additions><!--THIS element in domlib CONTAINS ESCAPED HTML, thus it is not possible to reliably extract the information about pages. Please watch out.-->
+
+                     <handNote xml:id="h2" script="Ethiopic">
+                       <locus from="19ra" to="20ra"/>
+                         <seg type="script">First half of the nineteenth-century raqiq script.</seg>
+                       <seg type="ink">Black; red.</seg>
+                       <seg type="rubrication">Incipit of the text; elements of the punctuation signs.</seg>
+                       <desc>Careful but irregular; by an unskilled scribe. Broad, upright script. Rectangular letter
+                       shapes.</desc>
+                       </handNote>
+                        </handDesc>
+
+                  <additions>
                      <list>
                         <item xml:id="a1">
-                           <locus target="#1va">f. 1va</locus>
-                           <desc>-vb: A miracle of Gabra Manfas Qǝddus.</desc>
-                           <quote xml:lang="gez"/>
-                           <quote xml:lang="en"/>
-                           <note> Written in a crude hand. </note>
+                           <locus target="#1va"/>
+                           <desc type="GuestText"><title ref="LIT3977Taammer">One miracle of Gabra Manfas Qǝddus.</title></desc>
+                           <quote xml:lang="gez">ተአምሪሁ፡ ለአቡነ፡ ገብረ፡ መንፈስ፡ ቅዱስ፡ ጸሎቱ፡ ወበረከቱ፡ የሀሉ፡ ምስለ፡ ገብሩ፡
+                             ተስፋ፡ ማርያም፡ ለዓለ፡ ወነበረ፡ ፩ነዳይ፡ ዘአልቦቱ፡ ጥሪት፡ ዘእንበለ፡ አሐዱ፡ ዶርሖ፡</quote>
+                           <quote xml:lang="en"></quote>
+                           <note>For the edition, s. <bibl><ptr target="bm:Marrassini2003GabraManfasQeddusTextus"/></bibl>
+                           and <bibl><ptr target="bm:Marrassini2003GabraManfasQeddusVersio"/>
+                           <citedRange unit="chapter">“Miracolo xii”</citedRange></bibl>.
+                           Starting from <ref target="#1vb"/>, l.5 the scribe (<ref target="#h1">Hand 1</ref>) used a different,
+                           thinner pen and wrote in smaller script.
+                         </note>
                         </item>
-                        <item xml:id="a2">
```

Comment:
I believe this addition got lost?
(NB locus from="18vb" to "19ra")

Reply:
Before, they both were one "additio". In fact, I do not know what it is. Magic? Writing exercise?